HACKING.rst

```diff
@@ -302,6 +309,7 @@ The following guidelines should be followed:
 .. _TestPrependBaseCommands: https://github.com/canonical/cloud-init/blob/master/cloudinit/tests/test_subp.py#L9
 .. _assertion introspection: https://docs.pytest.org/en/latest/assert.html
 .. _pytest 3.0: https://docs.pytest.org/en/latest/changelog.html#id1093
+.. _pytest.param: https://docs.pytest.org/en/latest/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions
```

Comment:
Should we link here instead? https://docs.pytest.org/en/latest/reference.html#pytest-param

Because of the link, I initially thought this was saying we couldn't use parametrization at all, but instead it's limited to that specific function. Not a super strong opinion, so +1 whatever you decide.

Reply:
Yep, and I'll update the start of the bullet to read "The `pytest.param` function cannot be used." for further clarification. Good catch!